optuna/logging.py

```diff
@@ -127,18 +127,32 @@ def disable_default_handler():
 
     Example:
 
-        Stop and then resume logging to standard output.
+        Stop and then resume logging to stderr.
 
-        .. code::
+        .. testsetup::
+
+            def objective(trial):
+                x = trial.suggest_uniform('x', -100, 100)
+                y = trial.suggest_categorical('y', [-1, 0, 1])
+                return x ** 2 + y
+
+        .. testcode::
+
+            import optuna
+
+            study = optuna.create_study()
+
+            # There are no logs in stderr.
+            optuna.logging.disable_default_handler()
+            study.optimize(objective, n_trials=10)
+
+            # There are logs in stderr.
+            optuna.logging.enable_default_handler()
+            study.optimize(objective, n_trials=10)
+            # [I 2020-02-23 17:00:54,314] Finished trial#10 resulted in value: ...
```

Comment:
These lines outputted as stderr, and it is difficult test it. So, I decided to write them as comments 🤔